Add Unmanic

drag0n141 opened the pull request.

drag0n141 committed:
Add Unmanic

doob187 committed:
removed folder /compose

not needed anymore

doob187 commented:
massiv change 
removed /compose folder 
removed /help ( since we have the wiki running )

doob187 committed:
add sublayout

add sublayout for traefik
add part to show the user the running traefik before traefik configs get overwritten

doob187 committed:
Update ubuntu.sh

doob187 reviewed (approved):
all viewed and tested

doob187 committed:
Create tls.yml

doob187 committed:
Merge pull request #8 from dockserver/Tls

Create tls.yml
